### PR TITLE
feat: implemented generator and renderer for python

### LINKED ISSUE #49: Implement Python generator and renderers
Implement Python generator and renderers like in TS or Java.

Generator should have possibility to define presets for `class`, `enum` type.

Additional: create presets for serialisation/converters from json.

### PR DESCRIPTION
**Pull request by mahakporwal02**
Fixes #49 
I have implemented `ClassRenderer.ts` right now.
I'll implement `EnumRenderer.ts` once my current implementation of `ClassRenderer.ts` has been approved.

### CODE REVIEW

**Review by jonaslagoni — CHANGES_REQUESTED** [review]
The existing code looks great, awesome 🎉! Only one small coding smell from sonar 🙂 

I see I forgot all about the tests and documentation that was needed to change, sorry!

Do you mind adding the following tests?
- We need to ensure that the file generator for python works https://github.com/asyncapi/modelina/blob/master/test/generators/java/JavaFileGenerator.spec.ts
- We need to test the generator for specific cases (where it would not make sense to make examples) https://github.com/asyncapi/modelina/blob/master/test/generators/java/JavaGenerator.spec.ts. Here we primarily need: 
  - https://github.com/asyncapi/modelina/blob/810d607cb6ada153f8d4e884a1d864350e7e2845/test/generators/java/JavaGenerator.spec.ts#L12
  - https://github.com/asyncapi/modelina/blob/810d607cb6ada153f8d4e884a1d864350e7e2845/test/generators/java/JavaGenerator.spec.ts#L454
- We need to make sure the renderer's functions work as a unit test. See https://github.com/asyncapi/modelina/blob/master/test/generators/typescript/TypeScriptRenderer.spec.ts

We also need to add some documentation, so that people know the python generator exists 🙂 
- We need to add python specific language documentation, it will just be an empty one like this https://github.com/asyncapi/modelina/blob/master/docs/languages/Go.md
- Add your example to the usage documentation https://github.com/asyncapi/modelina/blob/master/docs/usage.md
- Do you mind removing the language list under https://github.com/asyncapi/modelina/blob/master/docs/advanced.md#generate-models-to-separate-files. As all generators support this.
- Add python to the list in the main readme https://github.com/asyncapi/modelina/blob/master/README.md#features

File: examples/generate-python-models/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Python data models
+
+A basic example of how to use Modelina and output a Pyton data model.
```
Comment (liquidautumn) [inline examples/generate-python-models/README.md:3]: typo in Python

**Comment by liquidautumn** [comment]
Chiming in to express gratitude for the great work and confirm this is much needed feature. Currently I'm generating ts models with modelina and then use it as a source for quicktype to generate python models. It works, but adds plenty of extra dependencies, looking forward to this pr to be merged!

**Comment by mahakporwal02** [comment]
> Chiming in to express gratitude for the great work and confirm this is much needed feature. Currently I'm generating ts models with modelina and then use it as a source for quicktype to generate python models. It works, but adds plenty of extra dependencies, looking forward to this pr to be merged!

Thank you, I'll complete this PR ASAP :)

**Comment by niraveen** [comment]
Any update on this pull request?

**Comment by jonaslagoni** [comment]
@mahakporwal02 any update from your side, or is it okay if I start moving the PR forward?

**Comment by jonaslagoni** [comment]
Continued in https://github.com/asyncapi/modelina/pull/863

**Pull request closed without merging**